src/components/MetricChart/MetricChart.js

```diff
@@ -119,19 +119,36 @@ const GenericMetricChart = ({ chartConfig, showGrid }) => {
       })
     }
 
+    if (chartConfig.gradient) {
+      const canvasHeight = chartInstance.current.height
+      const showGrid = canvasHeight > 70 ? true : false
```

Comment:
redundant ternary

Reply:
entire logic is redundant because the grid is shown for both cases

Reply:
done